chpl/chpl-api/e2e/script/newman-run.js

```diff
@@ -118,13 +163,17 @@ const searchviewControllerTests = {
 const jobs = [
   cb => newman.run(acbControllerTests,cb),
   cb => newman.run(apiKeyControllerTests,cb),
-  cb => newman.run(certifiedProductControllerTests,cb),
+  cb => newman.run(certifiedProductControllerDataTests2015listings,cb),
+  cb => newman.run(certifiedProductControllerDataTests2014listings,cb),
+  cb => newman.run(certifiedProductControllerNoDataTests,cb),
   cb => newman.run(developerControllerDataTests,cb),
   cb => newman.run(developerControllerNoDataTests,cb),
+  cb => newman.run(jobControllerTests,cb),
```

Comment:
I don't see jobControllerTests added in this PR, not sure why is it added here?

Reply:
'job' and 'scheduler' controller tests for ONC Staff were added as part of OCD-3584. But those are not in newman-run.js latest file . So added those in this PR.